src/scss/components/_menu-points.scss

```
@@ -12,18 +12,22 @@
   grid-row-gap: var(--title-gap);
   grid-template-areas: 'icon heading' 'empty keywords';
 
-  &--title {
-    grid-area: heading;
-    align-self: center;
-  }
-  .svg,
+  .svg-wrapper,
   img {
     grid-area: icon;
     align-self: center;
```

```suggestion
    align-self: start;
```
Should align to top. Check with design that placing is ok